mittab/libs/data_import/export_xls_files.py

```diff
@@ -0,0 +1,220 @@
+from django.contrib.auth.decorators import permission_required
+from django.http import HttpResponse
+from xlwt import Workbook
+
+from mittab.apps.tab.models import Team, Judge, Room, Debater
+from mittab.libs import tab_logic
+
+
+def _create_vn_str(novice_status):
+    """Creates varsity-novice status string from the integer pseudo-enum used by the model"""
+    if novice_status is 0: return "varsity"
+    if novice_status is 1: return "novice"
+    return novice_status
+
+
+@permission_required('tab.tab_settings.can_change', login_url="/403/")
```

Comment:
No need to require this permission since it doesn't write anything to the database

Reply:
How I saw this was that one would probably not prefer being able to straight generate such data from something like a URL hack, which I believe (unsure whether entirely factual) is possible via URL hack